Tribler/community/tunnel/main.py

```diff
@@ -143,7 +143,8 @@ def start_community():
 
             self.session.set_anon_proxy_settings(
                 2, ("127.0.0.1", self.session.get_tunnel_community_socks5_listen_ports()))
-
+            if introduce_port:
+                self.community.add_discovered_candidate(Candidate((self.dispersy._wan_address[0], introduce_port), tunnel=False))
```

Comment:
When running the main.py on e.g. port 8847, this causes dispersy to discover both 127.0.0.1:8847 and later also 10.0.0.25:8847 (or whatever ip). The same candidate is twice in the list of verified candidates and causing crypto problems (same hop used twice in circuit). That's why I'm using the dispersy wan address here.